docker/courtlistener/docker-compose.yml

```diff
@@ -72,7 +78,13 @@ services:
 
   cl-django:
     container_name: cl-django
-    image: ${DJANGO_DOCKER_IMAGE:-freelawproject/courtlistener:latest-web-dev}
+    build:
+      context: "../../"
+      dockerfile: "./docker/django/Dockerfile"
+      args:
+        BUILD_ENV: dev
+      target: web-dev
+    image: freelawproject/courtlistener:latest-web-dev
```

Comment:
@mlissner I wasn't 100% sure what to label/name the images; LMK if these need to be changed for `cl-django` and `cl-celery`.

Reply:
Two things (that we can work out over Slack):

1. `image` might be redundant? What would be necessary for us to rely on `build` entirely.
2. `DJANGO_DOCKER_IMAGE` is removed here, but it persists in `.github/workflows/tests.yml`. There should be a corresponding change?

Perhaps reliance on `build` and removal of `image` could mean that `latest-*-dev` images don’t need to exist and some more thorough simplification can be done.